react/game/src/app/screens/Game/index.js

```diff
@@ -1,73 +1,38 @@
 import React, { Component } from 'react';
+import { arrayOf, bool, string, number, shape, func } from 'prop-types';
+import { connect } from 'react-redux';
 
-import { winLines } from '../../../constants/game';
+import { calculateWinner } from '../../../utils/game';
+import { actionCreators } from '../../../redux/game/actions';
```

Comment:
I'd recommend setting up these paths in webpack so you can do absolute imports. 👍

Reply:
Excelent 👌🏼